### PR TITLE
Wqp 1301

### PR DESCRIPTION
**Pull request by mbucknell**
Updated for postgres

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,43 @@
 # schema\-biodata
 
-Liquibase scripts for creating the WQP biodata database schema objects. They are used for the Water Quality Portal (WQP).
 
-The scripts are executed via a Jenkins job.
+Liquibase scripts for creating the BIODATA\_STEWARDS database schema objects in a Postgres database.
```
Comment (dsteinich) [inline README.md]: \_STEWARDS is not applicable.

File: liquibase/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM openjdk:11.0.1
+
+MAINTAINER Mary Bucknell <mbucknell@usgs.gov
```
Comment (dsteinich) [inline liquibase/Dockerfile]: MAINTAINER has been deprecated & replaced with:

LABEL maintainer="gs-w_eto_eb_federal_employees@usgs.gov"

File: liquibase/dbInit/1_run_preload_liquibase.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	-DPOSTGRES_PASSWORD=${POSTGRES_PASSWORD} \
+	-DBIODATA_DATABASE_NAME=${BIODATA_DATABASE_NAME} \
+	-DBIODATA_OWNER=${BIODATA_OWNER} \
+	-DBIODATA_OWNER_PASSWORD=${BIODATA_OWNER_PASSWORD} \
+	-DBIODATA_SCHEMA_NAME=${BIODATA_SCHEMA_NAME} \
+	-DWQP_CORE_OWNER=${WQP_CORE_OWNER} \
+	-DWQP_CORE_OWNER_PASSWORD=${WQP_CORE_OWNER_PASSWORD} \
+	 > $LIQUIBASE_HOME/logs/liquibasePostgres.log
+
+# postgres to pubs db scripts
```
Comment (dsteinich) [inline liquibase/dbInit/1_run_preload_liquibase.sh]: Copy paste issue - this is not pubs.

**Comment by mbucknell** [comment]
Made the requested changes.

**Pull request merged**